pages/lobby/[id].js

```diff
@@ -31,6 +31,24 @@ const useQueuedMessage = (addMessage) => {
     return setQueuedMessage;
 };
 
+const useShareLink = () => {
+    const [shareClicked, setSharedClicked] = React.useState(false);
+
+    React.useEffect(() => {
+        if (shareClicked) {
+            const timeout = setTimeout(() => setSharedClicked(false), 2000);
+            return () => clearTimeout(timeout);
+        }
+    }, [shareClicked]);
+    const onShareLink = () => {
+        navigator.clipboard.writeText(window.location.href).then(() => {
```

Comment:
.catch(err => {
    // This can happen if the user denies clipboard permissions:
    console.error('Could not copy text: ', err);
  });